RELEASE_NOTES.md

```diff
@@ -0,0 +1,45 @@
+# cddl-gen v. 0.3.0
+
+## Improvements:
+
+### CDDL language
+ * Add more support for hexadecimal/octal/binary numbers in CDDL. E.g. in size ranges or value limits
+ * Add support for ranges with three dots in CDDL files
+ * Change CDDL quantifier syntax from x**y to x*y
+ * Add support for $$ and $
+ * Add support for the .bits operator.
+
+### Command line
+ * Change command line name from cddl_gen to cddl-gen
+ * Add --dq as a shorthand for --default-max-qty
```

Comment:
Would also suggest inline markdown syntax here for `cddl_gen`, `cddl-gen`, `--dq`, and `--default-max-qty`.

Same goes for other code bits like function calls in the file, like `target_cddl_source()`. File names can also look clearer sometimes, but no strong opinion on that here.